### DISCUSSION TITLE
Add ability to change the default calendar

### DISCUSSION THREAD

**Discussion by asmagen** [davish/obsidian-full-calendar#420] [Feature Requests]
**Describe the bug**
There is nowhere to change the default calendar. Changing the "defaultCalendar" parameter in data.json doesn't change anything.

**Comment by davish** [comment]
This is not a fully implemented feature, but it is not a bug. Changing this to a feature request.

**Comment by davish** [comment]
The current workaround is to delete all calendars and re-add them so that the desired default calendar is first.
